### PR TITLE
Merge 18.3.0.1 beta with editorialized release notes into `develop`

### PR DESCRIPTION
**Pull request by mokagio**
I shipped a new beta so we can test the changes from https://github.com/wordpress-mobile/gutenberg-mobile/pull/3952 which depend on the translation of a new string, as requested by @startuptester.

### CODE REVIEW

File: WordPress/Resources/bg.lproj/Localizable.strings [hunk 1]
```diff
@@ -1503,6 +1503,9 @@ translators: Block name. %s: The localized block name */
 /* Title displayed for selection of custom app icons that have colorful backgrounds. */
 "Colorful backgrounds" = "Colorful backgrounds";
 
+/* translators: %d: column index. */
+"Column %d" = "Column %d";
```
Comment (mokagio) [inline WordPress/Resources/bg.lproj/Localizable.strings:1507]: This is the particular string that we needed in the localizations 😉 as per https://github.com/wordpress-mobile/gutenberg-mobile/pull/3952

File: WordPress/Resources/es.lproj/Localizable.strings [hunk 2]
```diff
@@ -1503,6 +1503,9 @@ translators: Block name. %s: The localized block name */
 /* Title displayed for selection of custom app icons that have colorful backgrounds. */
 "Colorful backgrounds" = "Fondos coloridos";
 
+/* translators: %d: column index. */
+"Column %d" = "Columna %d";
```
Comment (mokagio) [inline WordPress/Resources/es.lproj/Localizable.strings:1507]: @startuptester as you can see here, the Spanish version has been translated, might be good to test against.

**Review by mokagio — COMMENTED** [review]
- [x] Version update in `.xcconfig`
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] `AppStoreStrings.po` changed with editorialized release notes

File: WordPress/Jetpack/Resources/AppStoreStrings.po [hunk 3]
```diff
@@ -38,16 +38,11 @@ msgctxt "app_store_keywords"
...
 "\n"
-"- You have even more control of your site comments — when you edit them you can now also adjust the author’s name, email address, and web address.\n"
-"We have made a couple of handy tweaks to the Block editor. The Embed block now gives you a sneak preview when you add something from YouTube or Twitter (more providers to come). Also, when you go to insert a block and nothing is found, there is now a no results view.\n"
-"\n"
-"- If you go to Post Settings > Post Format, you will find the “Standard” option still at the top, but the other options are now neatly alphabetized below. (Options will vary with themes!)\n"
-"\n"
-"- And one more quick one: Refreshing your site is now super easy — just pull to refresh from the My Site screen!\n"
+"-Various improvements and fixes: We’ve streamlined your My Site menu, changing “Blog Posts” to “Posts” and “Site Pages” to “Pages.” (This has not affected your post and page content — we promise!) We also fixed a bug that prevented you from following a conversation by email through a post’s comments.\n"
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/AppStoreStrings.po]: nit: no space after the `-` dashes?

File: WordPress/Resources/bg.lproj/Localizable.strings [hunk 4]
```diff
@@ -5198,7 +5232,8 @@ translators: Block name. %s: The localized block name */
 /* All Time Stats 'Posts' label
    Noun. Title. Links to the blog's Posts screen.
    Stats Posts Label
-   Title of a Reader tab showing Posts matching a user's search query */
+   Title of a Reader tab showing Posts matching a user's search query
+   Title of the screen showing the list of posts for a blog. */
```
Comment (AliSoftware) [inline WordPress/Resources/bg.lproj/Localizable.strings:5236]: The number of different contexts for this same copy is a bit scary, localization-wize 😅  (but we don't have a solution for that yet anyway, so 🤷 )

**Review by AliSoftware — APPROVED** [review]
Changes LGTM and the two different Editorial copies (WP vs JP) match the Editorial post.

Left a nit though in case you want to add spaces after "bullet-point" dashes.

File: WordPress/Jetpack/Resources/AppStoreStrings.po (hunk 3 above)
Comment (mokagio) [reply WordPress/Jetpack/Resources/AppStoreStrings.po]: This is how the copy was in the editorial post 🤷‍♂️ I don't have a strong opinion about the stylistic effect so I just kept it that way.

**Pull request merged**